modules/vsphere-anti-affinity.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-restricted-networks-vsphere.adoc
+// * installing/installing_vsphere/installing-vsphere-network-customizations.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
+
+:_content-type: PROCEDURE
+[id="anti-affinity-vsphere_{context}"]
+= Configuring vSphere DRS anti-affinity rules for control plane nodes
+
+vSphere Distributed Resource Scheduler (DRS) anti-affinity rules can be configured to support higher availability of OpenShift Control Plane nodes so that the vSphere Virtual Machines for these nodes are not scheduled to the same vSphere Host. 
+
+[IMPORTANT]
+====
+* The following information applies to compute DRS only and does not apply to storage DRS.
+
+* The `govc` command is an open-source command available from VMware; it is not available from Red Hat. `govc` is not supported by the Red Hat support. 
+
+* Instructions for downloading and installing 'govc' are found on the VMware documentation website.
+==== 
+
+// https://docs.vmware.com/en/VMware-Telco-Cloud-Operations/1.4.0/deployment-guide-140/GUID-5249E662-D792-4A1A-93E6-CF331552364C.html#:~:text=Govc%20is%20an%20open%20source,operations%20on%20the%20target%20vCenter.
+
+.Example command
+
+[source,terminal]
+----
+# govc cluster.rule.create \
+  -name openshift4-control-plane-group \
+  -dc MyDatacenter -cluster MyCluster \
+  -enable \
+  -anti-affinity master-0 master-1 master-2
+----
+
+After creating the rule, your control plane nodes will be automatically migrated by vSphere so they are not running on the same hosts. This might take some time while vSphere reconciles the new rule. Successful command completion is shown below:
```

Comment:
IBMSG recommends against using directions (i.e., "shown below). This mainly has to do with screen readers/non-traditional readers. So maybe we could reword it: 

"Successful command completion is shown in the following procedure." 

I also don't think this would get a colon, since it doesn't go directly into the procedure.

Reply:
Thank you! Done!